process/elasticproc/transactions/smartContractResultsProcessor.go

```diff
@@ -90,22 +84,18 @@ func (proc *smartContractResultsProcessor) processSCRsFromMiniblock(
 
 	indexerSCRs := make([]*indexerData.ScResult, 0, len(mb.TxHashes))
 	for _, scrHash := range mb.TxHashes {
-		scrHandler, ok := scrs[string(scrHash)]
+		scrInfo, ok := scrs[hex.EncodeToString(scrHash)]
```

Comment:
maybe extract it into a variable, since it's being used also for logging and delete

Reply:
done.